src/extension.ts

```diff
@@ -1,5 +1,6 @@
 import * as vscode from 'vscode';
 import { OSUtil } from '@arcsine/screen-recorder/lib/os';
+import { LiveShare } from 'vsls';
```

Comment:
This is simply importing the interface for typing purposes. This code doesn't emit anything when compiled, and the actual module import is done dynamically below.